build/bootstrap/bootstrap-debian-10.sh

```diff
@@ -31,15 +35,15 @@ ninja
 ninja install
 
 # The latest native-version of gtest on debian10 currently has a bug where CMakeLists doesn't declare an install target, causing 'make install' to fail
-# Clone from source and use release-1.10.0 instead, since gtest is a source package anyways 
+# Clone from source and use release-1.10.0 instead, since gtest is a source package anyways
 mkdir /tmp/gtest
 cd /tmp/gtest
 git clone https://github.com/google/googletest.git .
 git checkout release-1.10.0
-mkdir cmake 
+mkdir cmake
 cd cmake
 cmake /tmp/gtest
-make 
-make install 
+make
+make install
 
 echo "Finished bootstrapping"
```

Comment:
Something I've been thinking about based on other boostrap scripts is to have all of these consolidated into a single bootstrap script, where platform is automatically detected. That way we can deduplicate some of the script commands to install things like python, make, etc.

I think based on what I've seen, it's also fairly standard to have the bootstrap scripts do a build of whatever the repo is before completing to verify things work correctly.

Totally unrelated to this PR though, I'll open a new task for the above:
[https://microsoft.visualstudio.com/OS/_workitems/edit/32499146/?workitem=33759607](https://microsoft.visualstudio.com/OS/_workitems/edit/32499146/?workitem=33759607)

Reply:
Yes, de-duplicate and a test build sounds great.